Makefile

```diff
@@ -51,7 +51,7 @@ _repos/tutes:
 _repos/tutes/%.md: _repos/sel4proj/sel4-tutorials/tutorials/% _repos/tutes
 	PYTHONPATH=_repos/sel4/capdl/python-capdl-tool _repos/sel4proj/sel4-tutorials/template.py --docsite --out-dir _repos/tutes --tut-file $</$(@F)
 
-TUTORIALS:= $(filter-out index.md,$(notdir $(wildcard Tutorials/*.md)))
+TUTORIALS:= $(filter-out index.md overview.md setting-up.md get-the-tutorials.md,$(notdir $(wildcard Tutorials/seL4Kernel/*.md Tutorials/CAmkES/*.md Tutorials/DynamicLibraries/*.md Tutorials/MCS/*.md)))
```

Comment:
Something about the path `seL4Kernel` bugs me :-). It should be either `seL4` or `kernel`, but not `seL4Kernel`.

Reply:
Slightly more content-oriented -- should we really make a new directory structure here? Would it make sense to keep the directory names of the `sel4-tutorials` repo, or does the structure here get translated directly into navigation structure on the site?

Reply:
@lsf37 

> Something about the path `seL4Kernel` bugs me :-). It should be either `seL4` or `kernel`, but not `seL4Kernel`.

What about `mechanisms` since these are the `mechanisms` tutorials?

Reply:
`mechanisms` doesn't really say what the mechanisms apply to. Is `seL4` not good?

Reply:
I got `mechanisms` from the current tutorials overview page https://docs.sel4.systems/Tutorials/, from the **seL4 mechanisms tutorials** section.

But yeah, `seL4` is more specific.

Reply:
> Slightly more content-oriented -- should we really make a new directory structure here? Would it make sense to keep the directory names of the `sel4-tutorials` repo, or does the structure here get translated directly into navigation structure on the site?

Fair point. I'm working on reverting to 
```
TUTORIALS:= $(filter-out index.md,$(notdir $(wildcard Tutorials/*.md)))
```
but it's taking a while to fix all the broken links.